### DISCUSSION TITLE
leetcode-study 프로젝트에서 코드 실행 관련

### DISCUSSION THREAD

**Discussion by EcoFriendlyAppleSu** [DaleStudy/leetcode-study#669] [자료공유]
leetcode-study 프로젝트에서 작성한 코드를 실행하려고 하면, 이전 기수분들의 코드와 충돌이 발생하여 컴파일 오류가 발생합니다. (Intellij IDE 환경)

코드 실행은 leetcode 사이트 혹은 개인 IDE에서 진행하고 풀이된 코드만 leetcode-study 프로젝트에 반영하면 될 것 같습니다.

+) 혹시 leetcode-study 프로젝트에서 직접 코드를 실행해야 하는지 고민하시는 분들을 위해 안내드립니다.

<img width="271" alt="Kotlin_not_configured" src="https://github.com/user-attachments/assets/6a2102d0-82e0-4623-b7f4-fffd6abf9813">

**Comment by DaleSeo** [comment]
Java나 Kotlin 사용자에게 유용하겠네요. 꿀팁 공유해주셔서 감사합니다!

**Reply by renovizee** [reply]
Java나 Kotlin분들인 fork 받은 프로젝트에서 어떻게 실행하셨는지 궁금해서 찾아보고 있었는데 다른 곳에서 하셨군요. 내용 공유해주셔서 감사합니다.
